### ISSUE TITLE
Failed to open tile message

### ISSUE THREAD

**Issue by ldodds** [cgutteridge/geocraft#11]
```
failed to open /.../geocraft/var/tiles/19_258686_174660.png: No such file or directory
Can't use an undefined value as an ARRAY reference at /.../geocraft/lib/Minecraft/MapTiles.pm line 60.

```

Is this down to a missing area of LIDAR?

**Comment by cgutteridge** [comment]
Nope. That's an open streetmap tile that's not there. Why oh why is it 
/.../ ???

On 07/07/2016 20:26, Leigh Dodds wrote:

> |failed to open /.../geocraft/var/tiles/19_258686_174660.png: No such 
> file or directory Can't use an undefined value as an ARRAY reference 
> at /.../geocraft/lib/Minecraft/MapTiles.pm line 60. |
> 
> Is this down to a missing area of LIDAR?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub 
> https://github.com/cgutteridge/geocraft/issues/11, or mute the 
> thread 
> https://github.com/notifications/unsubscribe/AAfK65vKHFlOEhwLmTZNir58Z4AxMxMVks5qTVLRgaJpZM4JHarf.Web 
> Bug from 
> https://github.com/notifications/beacon/AAfK63jN90ML3UWeGQMvbrwf5YbYkI7_ks5qTVLRgaJpZM4JHarf.gif

## 

Christopher Gutteridge -- http://users.ecs.soton.ac.uk/cjg

University of Southampton Open Data Service: http://data.southampton.ac.uk/
You should read the ECS Web Team blog: http://blogs.ecs.soton.ac.uk/webteam/

**Comment by ldodds** [comment]
I just removed the local bits of my path :)

**Comment by amyworrall** [comment]
I've had a variation on this, where the OSM server returns a 404. It usually works on the second try, so my workaround is to manually remove the tile, open the OSM URL in my browser, download it myself, and save it with the correct file name.
